### ISSUE TITLE
Split screens overlap with Nouveau driver

### ISSUE THREAD

**Issue by supertuxkart-sourceforge-migration** [supertuxkart/stk-code#661]
**Author: mungewell**

Hi all,
moved my machine over to the Nouveau driver (from Nvidia propriatory driver) and noticed that the split screens no longer function properly.

In a 2 player game the top screen is displayed full screen, and the lower screen is not seen (except a few things like the 'rescue bird' show through).

Exactly the same issue was seen with SpeedDreams, their bug report and patch here:
https://sourceforge.net/apps/trac/speed-dreams/ticket/400

Basically you need a scissor to limit the drawing extremes of the split screen.

Cheers,
Simon.

Migrated-From: https://sourceforge.net/apps/trac/supertuxkart/ticket/605

**Comment by hiker** [comment]
Considering that the patch appears simple, can we apply this?

**Comment by auriamg** [comment]
I actually wonder if this is still an issue, we have received no recent reports of this

**Comment by Flakebi** [comment]
That's because most of the people with an nvidia gpu use the proprietary driver instead of nouveau.

**Comment by deveee** [comment]
Can someone check if this is still an issue with current drivers? I'm still not able to properly configure nouveau drivers on my laptop :( And this patch is relatively easy to apply.

**Comment by deveee** [comment]
I checked it with current nouveau drivers. STK works fine, also in multiplayer mode, so I think that this ticket can be closed.
